roles/insights/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: internal_host_started_processing_role
+  set_fact:
+    internal_host_started_processing_role: "insights"
+
+- name: gather insights.machine-id
+  raw: if [ -f /etc/insights-client/machine-id ] ; then cat /etc/insights-client/machine-id; fi || if [ -f /etc/redhat-access-insights ] ; then cat /etc/redhat-access-insights/machine-id; fi
+  register: internal_cat_insights_client_id
+  ignore_errors: yes
+
+- name: set instnum
```

Comment:
change this name to `extract insights client id from internal_get_insights_client_id`